modules/ank/arrow-ank/build.gradle

```diff
@@ -1,9 +1,10 @@
 apply plugin: 'kotlin'
```

Comment:
Is this a refactor or a merge? O.O

Reply:
Too fast again :)
Explained this below.